crates/node/Cargo.toml

```diff
@@ -9,24 +9,27 @@ license.workspace = true
 [dependencies]
 borsh.workspace = true
 camino = { workspace = true, features = ["serde1"] }
-clap = { workspace = true, features = ["derive"] }
+clap.workspace = true
```

Comment:
why was this removed? I'm pretty sure we use this for interactive CLI

Reply:
clippy told me it's unused, but true, I see it's used in the interactive CLI, I'll bring it back

Reply:
this was marked as resolved?